globals/utilities.php

```diff
@@ -198,28 +199,46 @@ function neve_cart_icon( $echo = false, $size = 15, $cart_icon = '', $icon_custo
 /**
  * Search Icon
  *
- * @param bool $is_link should be wrapped in A tag.
- * @param bool $echo should be echoed.
- * @param int  $size icon size.
- * @param bool $amp_ready Should we add the AMP binding.
+ * @param bool         $is_link should be wrapped in A tag.
+ * @param bool         $echo should be echoed.
+ * @param int          $size icon size.
+ * @param bool         $amp_ready Should we add the AMP binding.
+ * @param false|string $context Represents where the search icon is being used.
  *
  * @return string|null
  */
-function neve_search_icon( $is_link = false, $echo = false, $size = 15, $amp_ready = false ) {
+function neve_search_icon( $is_link = false, $echo = false, $size = 15, $amp_ready = false, $context = false ) {
+
+	$icon_type = SearchIconButton::DEFAULT_ICON;
+
+	if ( $context === 'hfg' ) {
+		$hfg_icon_type = \HFG\component_setting( SearchIconButton::ICON_TYPE );
```

Comment:
The control is added like this:
```
$sm::get_instance()->add(
			[
				'id'                 => self::ICON_TYPE,
				'group'              => $this->component_id,
				'tab'                => SettingsManager::TAB_STYLE,
				'transport'          => 'refresh',
				'label'              => __( 'Icon', 'neve' ),
				'default'            => self::DEFAULT_ICON,
				'sanitize_callback'  => 'sanitize_text_field',
				'type'               => '\Neve\Customizer\Controls\React\Radio_Buttons',
				'section'            => $this->section,
				'conditional_header' => true,
				'options'            => [
					'is_for'          => 'search_icon',
					'show_labels'     => false,
					'active_callback' => [ $this, 'is_icon_mode_enabled' ],
				],
			]
		);
```
Should'nt we pass the default argument to` \HFG\component_setting` ?

Reply:
Nice catch, I've added now 🚀